test/e2e/testing-manifests/storage-csi/gce-pd/controller_ss.yaml

```diff
@@ -21,7 +21,8 @@ spec:
       serviceAccountName: csi-gce-pd-controller-sa
       containers:
         - name: csi-snapshotter
-          image: gcr.io/gke-release/csi-snapshotter:v2.1.1-gke.0
+          # TODO: Replace this with the gke image once available.
+          image: k8s.gcr.io/sig-storage/csi-snapshotter:v3.0.2
```

Comment:
Can you also update the pdcsi image to v1.0.1-gke.0? There were some fixes related to snapshots I think.

Reply:
Done